src/test/java/org/entando/plugins/pda/core/service/task/FakeTaskService.java

```diff
@@ -31,38 +34,36 @@
     public static final String TASK_PROCESS_ID_2 = "processId2";
     public static final String TASK_PROCESS_INSTANCE_ID_2 = "processInstanceId2";
 
-    private final List<Task> tasks = createTasks();
+    @Override
+    public PagedRestResponse<Task> list(Connection connection,
+            PagedListRequest restListRequest) {
+        return new PagedRestResponse<>(new PagedMetadata<>(restListRequest, createTasks()));
+    }
 
     @Override
-    public PagedRestResponse<Task> list(Connection connection, PagedListRequest restListRequest) {
-        return new PagedRestResponse<>(new PagedMetadata<>(restListRequest, tasks));
+    public Task get(Connection connection, String id) {
+        for (Task task : createTasks()) {
+            if (task.getId().equals(id))  {
+                return task;
+            }
+        }
+
+        throw new TaskNotFoundException();
     }
 
     private List<Task> createTasks() {
         List<Task> result = new ArrayList<>();
 
-        result.add(Task.builder()
+        result.add(FakeTask.builder()
```

Comment:
Can we rename from `FakeTask` to `MockTask`. I think it is a really good thing to have around but probably we don't want the word "fake" running around in the codebase

Reply:
Please see other comment.

I've also moved all of this `Fake` classes to the tests folder to make sure we don't accidentally use it in our production code.